packages/flutter/test/widgets/table_test.dart

```diff
@@ -348,7 +348,7 @@ void main() {
     expect(boxA2, isNotNull);
     expect(boxG2, isNotNull);
     expect(boxA1, equals(boxA2));
-    expect(boxG1, isNot(equals(boxG2)));
+    expect(boxG1, equals(equals(boxG2)));
```

Comment:
this seems concerning, there shouldn't be any logic changes right?

Reply:
https://github.com/flutter/flutter/pull/69670/files#r565986835

Reply:
aah, i understand now. this is literally the change this PR is about. Sorry I misunderstood the test before.